main.go

```diff
@@ -144,6 +144,7 @@ func main() {
 		log.Error("Failed to create SubnetManager: ", err)
 		os.Exit(1)
 	}
+	log.Info(fmt.Sprintf("Created subnet manager: %+v", sm))
```

Comment:
Did you forget this one in `main.go`? https://github.com/coreos/flannel/pull/717/files#diff-7ddfb3e035b42cd70649cc33393fe32cR147